airflow-core/src/airflow/serialization/serde.py

```diff
@@ -365,26 +365,23 @@ def _register():
     _stringifiers.clear()
 
     with Stats.timer("serde.load_serializers") as timer:
-        for _, name, _ in iter_namespace(airflow.serialization.serializers):
-            name = import_module(name)
-            for s in getattr(name, "serializers", ()):
-                if not isinstance(s, str):
-                    s = qualname(s)
+        for _, module_name, _ in iter_namespace(airflow.serialization.serializers):
+            name = import_module(module_name)
```

Comment:
```suggestion
            module = import_module(module_name)
```

I think it should be a module instead